packages/airnode-utilities/src/index.ts

```diff
@@ -5,3 +5,5 @@ export * from './utils/date-utils';
 export * from './utils/promise-utils';
 export * from './utils/string-utils';
 export * from './logging-types';
+export * from './utils/git';
```

Comment:
Minor thing, but I don't think we need a "utils" folder in the "airnode-utils" package. I think all of these could live in folders that explain their context/domain. Something like:

```sh
src/docker/
src/git/
src/logging/
src/promises/
src/strings/
```

What do you think?

Reply:
It will be done ser :D

Reply:
Please check :pray: :laughing: